media/com_media/js/EventBus.js

```diff
@@ -0,0 +1,100 @@
+(function (root, factory) {
+	if(typeof exports === 'object' && typeof module === 'object')
+		module.exports = factory();
+	else if(typeof define === 'function' && define.amd)
+		define("EventBus", [], factory);
+	else if(typeof exports === 'object')
+		exports["EventBus"] = factory();
+	else
+		root["EventBus"] = factory();
+})(this, function() {
+
+	var EventBusClass = {};
+	EventBusClass = function() {
+		this.listeners = {};
+	};
+	EventBusClass.prototype = {
+		addEventListener:function(type, callback, scope) {
```

Comment:
Either import the library as is or remove the amd, requirejs part. I guess the first, although I still don't see the need here